back/app/models/app_configuration.rb

```diff
@@ -43,7 +43,7 @@ module Settings
     extend CitizenLab::Mixins::SettingsSpecification
 
     def self.json_schema
-      settings_schema = core_settings_json_schema
+      settings_schema = core_settings_json_schema.deep_dup # deep_dup to avoid modifying core schema
```

Comment:
FYI: "If your object has instance variables that are hashes or arrays, those will not be deep_duped"

Not sure if this is an issue, but because of pressing issue thought I'd mention it ([reference](https://makandracards.com/makandra/478596-ruby-rails-clone-vs-dup-vs-deep_dup))

Reply:
It's just a hash, not a custom object, so I don't think we use any of its instance variables.